cmd/lean/lean.go

```diff
@@ -138,6 +143,29 @@ var lean = &cobra.Command{
 		}
 
 		fmt.Print(string(byt))
+
+		client, err := cisclient.NewClient(cisAddr)
+		if err != nil {
+			panic(err)
```

Comment:
`panic` is fine while we're developing, but we'll want to handle this a little more gracefully when we get closer to done.